conf/config-default.yaml

```diff
@@ -571,13 +571,13 @@ plugin_attr:          # Plugin attributes
       request_headers:              # Set the headers to include in requests to the OpenTelemetry collector.
         Authorization: token        # Set the authorization header to include an access token.
     batch_span_processor:
-      drop_on_queue_full: false     # Drop spans when the export queue is full.
-      max_queue_size: 1024          # Set the maximum size of the span export queue.
-      batch_timeout: 2              # Set the timeout for span batches to wait in the export queue before
+      drop_on_queue_full: true      # Drop spans when the export queue is full.
```

Comment:
Because the doc and the values in config-default.yaml are different, we need to align them by updating either the doc or the default config file.

The current apisix doc aligns its values with [opentelemetry-lua](https://github.com/yangxikun/opentelemetry-lua/blob/main/lib/opentelemetry/trace/batch_span_processor.lua#L106), which is the dependency used by the plugin. Therefore, it may be preferable to correct the default configuration file.

Reply:
the default values in apisix config file would have been set considering APISIX's use cases. Updating the doc would be better also because existing users might already be relying on the default values, changing them would lead to another effort being required during version upgrade, no?